### DISCUSSION TITLE
Its not working

### DISCUSSION THREAD

**Discussion by mrsuspotato5000** [3kh0/ext-remover#488] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
When i try to enter the script into a new page it does not work and i cannot use the webstorex thing so can i have some help?

**Comment by amelia-bartlett** [comment]


**Reply by mrsuspotato5000** [reply]
My school blocked Javascript commands so when I try to do the Java script bookmark nothing happens and the chromewebstorex thing it just blocks it with no 404 error page

**Comment by S-PScripts** [comment]
is this related to ltbeef?

thats an old exploit just use a new one

(necropost to answer)
